src/ralph/dns/views.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import logging
+from urllib.parse import urlencode, urljoin
+
+import requests
+from django.conf import settings
+from django.contrib import messages
+from django.forms import BaseFormSet, formset_factory
+from django.http import HttpResponseRedirect
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin.views.extra import RalphDetailView
+from ralph.dns.forms import DNSRecordForm, RecordType
+
+logger = logging.getLogger(__name__)
+
+
+def get_api_kwargs(url, data=None, headers=None):
```

Comment:
please move all ``*_api_*`` and ``*_dns_*`` method to class and separated module like ``dnsass.py``